CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - `ProofListIndex` and `ProofMapIndex` `root_hash` method has been renamed to
   `merkle_root`.
 
+#### exonum-configuration
+
+- `majority_count: Option<u16>` configuration parameter is introduced.
+  It allows increasing the number of votes for configuration proposal necessary for commit.
```

Comment:
Explanation is unclear. Please consider rephrasing as follows:
"Allows to increase the threshold amount of votes required to commit a new configuration proposal. By default the number of votes is set to `2/3` of total validators count."